### PR TITLE
Created parameter store resources for terraform folder

### PR DESCRIPTION
**Pull request by deniseddi**
To store WordPress secrets, created 4 key-value pairs. 
Built in Terraform. Named the  file with the resources `parameter_store.tf`

### CODE REVIEW

File: terraform/parameter_store.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+resource "aws_ssm_parameter" "wordpress-db-host" {
+  name        = "/wordpress/WORDPRESS_DB_HOST"
+  description = "The host parameter to be used by the container and DB"
```
Comment (kikobr82) [inline terraform/parameter_store.tf]: This value will only be used by the container

File: terraform/parameter_store.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+resource "aws_ssm_parameter" "wordpress-db-user" {
+  name        = "/wordpress/WORDPRESS_DB_USER"
+  description = "The user parameter to be used by the container and DB"
+  type        = "SecureString"
+  value       = "secret"
+}
+
+resource "aws_ssm_parameter" "wordpress-db-password" {
+  name        = "/wordpress/WORDPRESS_DB_PASSWORD"
+  description = "The password parameter to be used by the container and DB"
```
Comment (kikobr82) [inline terraform/parameter_store.tf]: This value will only be used by the container

File: terraform/parameter_store.tf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  type        = "SecureString"
+  value       = "secret"
+}
+
+resource "aws_ssm_parameter" "wordpress-db-name" {
+  name        = "/wordpress/WORDPRESS_DB_NAME"
+  description = "The name parameter to be used by the container and DB"
```
Comment (kikobr82) [inline terraform/parameter_store.tf]: This value will only be used by the container

File: terraform/parameter_store.tf [hunk 4]
```diff
@@ -0,0 +1,27 @@
+resource "aws_ssm_parameter" "wordpress-db-host" {
+  name        = "/wordpress/WORDPRESS_DB_HOST"
+  description = "The host parameter to be used by the container and DB"
+  type        = "SecureString"
+  value       = "secret"
+}
+
+resource "aws_ssm_parameter" "wordpress-db-user" {
+  name        = "/wordpress/WORDPRESS_DB_USER"
+  description = "The user parameter to be used by the container and DB"
```
Comment (kikobr82) [inline terraform/parameter_store.tf]: This value will only be used by the container

**Review by kikobr82 — COMMENTED** [review]
minor changes...

File: terraform/parameter_store.tf (hunk 1 above)
Comment (deniseddi) [reply terraform/parameter_store.tf]: I see, it will be used by the container to connect with the Database.

**Review by kikobr82 — APPROVED** [review]
lgtm

**Pull request merged**